code/API_definitions/knowledge-base-manage.yaml

```diff
@@ -313,18 +313,37 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorInfo'
+            allOf:
+              - $ref: '#/components/schemas/ErrorInfo'
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 400
+                  code:
+                    enum:
+                      - INVALID_DOCUMENT_FORMAT
+                      - MISSING_REQUIRED_PARAM
+                      - "{{SPECIFIC_CODE}}"
           examples:
-            InvalidFormat:
+            GENERIC_400_INVALID_DOCUMENT_FORMAT:
+              description: Invalid document format
               value:
                 status: 400
                 code: INVALID_DOCUMENT_FORMAT
                 message: "Invalid document format. Supported formats: doc/pdf/txt/csv"
-            MissingParameter:
+            GENERIC_400_MISSING_REQUIRED_PARAM:
+              description: Missing required parameter
               value:
                 status: 400
                 code: MISSING_REQUIRED_PARAM
                 message: "Required parameter 'name' is missing"
+            GENERIC_400_{{SPECIFIC_CODE}}:
```

Comment:
This is a placeholder in the CAMARA_common.yaml, to be replaced by your specific codes, in format {{API_NAME}}.{{SPECIFIC_CODE}}, e.g. QA_KNOWLEDGE_BASE.INVALID_DOCUMENT_FORMAT